### PR TITLE
opam-web.sh: append the redirect to the repo file, instead of replacing the entire file

### PR DESCRIPTION
**Pull request by hannesm**
//cc @kit-ty-kate -- this should do the trick, or shouldn't it?

### CODE REVIEW

File: bin/opam-web.sh [hunk 1]
```diff
@@ -22,11 +22,8 @@ mv opam-repository/* .
 mv opam-repository/.git .
 rm -rf opam-repository
 
-# Overwrite 'repo' file, and dispatch all non-standard versions
-cat <<EOF >repo
-opam-version: "2.0"
-browse: "https://${BASEURL}/pkg/"
-upstream: "https://github.com/ocaml/opam-repository/tree/master/"
+# Append to the 'repo' file, and dispatch all non-standard versions
+cat <<EOF >>repo
```
Comment (kit-ty-kate) [inline bin/opam-web.sh:26]:
```suggestion
sed -i 's/^browse: /d' repo
cat <<EOF >>repo
browse: "https://${BASEURL}/pkg/"
```

Comment (hannesm) [reply bin/opam-web.sh:27]: but why? In opam-repository the `browse` points to `https://opam.ocaml.org/pkg` (and aside, what is `browse` used for?)

Comment (kit-ty-kate) [reply bin/opam-web.sh:27]: > and aside, what is browse used for?

no idea. @rjbou do you know what it's used for?

Comment (kit-ty-kate) [reply bin/opam-web.sh:26]: I removed my commit

Comment (hannesm) [reply bin/opam-web.sh:26]: FWIW, https://opam.ocaml.org/pkg/ returns an empty page (404, Not found). In the opam sources, there's nothing using the `browse` from OpamFile.RepoSyntax. From the Manual.md: ``* <a id="repofield-browse">`browse: <string>`</a>:
  An URL where the users may browse available packages online
``

**Review by kit-ty-kate — APPROVED** [review]

**Comment by kit-ty-kate** [comment]
now live after testing it on staging.opam.ocaml.org (sorry for the delay i was at EMF)

**Pull request merged**